backend/back/src/grpc/grpc.service.ts

```diff
@@ -72,15 +77,18 @@ export class GrpcService implements OnModuleInit {
 
 	async onAction(data: JobData) {
 		this.logger.log(`Received action job data ${JSON.stringify(data, undefined, 2)}`);
+		await this.activityService.createActivityLogsForJobIdentifier("ran", data.identifier);
```

Comment:
Is this supposed to be displayed to the user without any form of formatting? If so please use the job name instead of the job identifier, otherwise lgtm